services/web/server/src/simcore_service_webserver/groups/models.py

```diff
@@ -1,6 +1,7 @@
 from ._utils import convert_groups_db_to_schema
 
 # Avoids pycln to remove import
+# mypy: disable-error-code=truthy-function
```

Comment:
maybe this at the top of the module is better like we do with pylint?

Reply:
ok